mautrix_signal/portal.py

```diff
@@ -945,6 +945,13 @@ async def handle_signal_message(
         else:
             self.log.debug(f"Didn't get event ID for {message.timestamp}")
 
+    async def handle_signal_kicked(self, user: u.User, sender: pu.Puppet) -> None:
+        self.log.debug(f"{user.mxid} was kicked by {sender.number} from {self.mxid}")
+        await sender.intent_for(self).send_notice(self.mxid, "Removed you from the chat.")
+        levels = await self.main_intent.get_power_levels(self.mxid)
+        levels.events_default = 50
+        await self.main_intent.set_power_levels(self.mxid, levels)
```

Comment:
This doesn't work for multi-user bridges, it should probably just actually kick the user